collector/systemd_linux.go

```diff
@@ -57,23 +60,36 @@ func NewSystemdCollector() (Collector, error) {
 		"Whether the system is operational (see 'systemctl is-system-running')",
 		nil, nil,
 	)
+	summaryDesc := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystem, "unit_state_summary"),
```

Comment:
I would suggest simply calling this `units`.  So the metric name would be `node_systemd_units`